tests/integration/targets/filter_counter/tasks/main.yml

```diff
@@ -26,7 +26,7 @@
   assert:
     that:
       - res is failed
-      - res.msg is match('Argument for community.general.counter must be a sequence')
+      - res.msg is search('Argument for community.general.counter must be a sequence')
```

Comment:
In `re` we have both, and the semantics are different. In here, are we changing the semantics,changing the name, or both?

Reply:
Errors during templating changed, so the actual error message is now part of a longer text. That's why `match` won't work anymore.